### PR TITLE
Feat message quote

### PR DESCRIPTION
**Pull request by hcfw007**
https://github.com/wechaty/wechaty/issues/2387

### CODE REVIEW

**Comment by windmemory** [comment]
To implement quote message operation, I think we need to update some of the `messageSend` mehotd in `puppet`.

**Comment by hcfw007** [comment]
> To implement quote message operation, I think we need to update some of the `messageSend` mehotd in `puppet`.

Yes, but that also requires the discussion on sayOptions to be finished because messageSendText also have a optional variable mentionIdList.

**Comment by windmemory** [comment]
typo fixed! haha

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "wechaty-puppet",
-  "version": "1.19.1",
+  "version": "1.19.2",
```
Comment (windmemory) [inline package.json:3]: Seems like this is a breaking change, shall we update the minor version?

Comment (hcfw007) [reply package.json:3]: I'm not sure. I think I managed to keep it compatible with previous versions.

**Pull request closed without merging**